README.md

```diff
@@ -1,95 +1,108 @@
-## ChoiceScript Library (cslib)
+<h1><img src="./res/logo/cslib_logo%20flat.png" alt="C.S.Lib logo" width="43" align="left"/> ChoiceScript Library</h1>
 
-The ChoiceScript library (cslib) is a collection of common, re-usable routines (i.e. functions) for use in development of ChoiceScript games. 
+![Release](https://img.shields.io/github/release/ChoicescriptIDE/cslib) ![GitHub](https://img.shields.io/github/license/ChoicescriptIDE/cslib) ![GitHub all releases](https://img.shields.io/github/downloads/ChoicescriptIDE/cslib/total) ![Discourse topics](https://img.shields.io/discourse/topics?server=https%3A%2F%2Fforum.choiceofgames.com%2F) 
+[![Build Status](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Factions-badge.atrox.dev%2FChoicescriptIDE%2Fcslib%2Fbadge%3Fref%3Dmain&style=flat)](https://actions-badge.atrox.dev/ChoicescriptIDE/cslib/goto?ref=main)
 
-cslib is written entirely in ChoiceScript itself, so is easy to understand, configure and use.
+The ChoiceScript Library (cslib) is a collection of common, re-usable routines (i.e. functions) for use in development of ChoiceScript games. It's an attempt to extend ChoiceScript's core functionalities using vanilla ChoiceScript only. It relies heavily on string parsing and string manipulation.
 
-The idea is to provide a community built (and driven) collection of routines that provide ChoiceScript with extended functionality and usage patterns. 
+[ChoiceScript](https://github.com/dfabulich/choicescript) is a simple programming language created by Dan Fabulich to aid non-programmers to write and code interactive fiction text-based gamebooks in the style of [Choose Your Own Adventure](https://en.wikipedia.org/wiki/Choose_Your_Own_Adventure). The language is proprietary and governed by its own [license](https://github.com/dfabulich/choicescript/blob/master/LICENSE.txt).
 
-- **Unblock Creation**
-    cslib can aid in ChoiceScript development by providing complicated functionality out of the box, or allowing certain code patterns to be written more concisely.
+The idea is to provide a community built (and driven) collection of routines, so [contribute](#-contributing) with ideas, code or feedback!
 
-- **Stimulate Learning**
-    cslib is written in ChoiceScript, so it's easy for any ChoiceScript author to read, understand, and learn from the cslib code. In time, they may even choose to contribute themselves.
+**Create, Learn and Experiment!**
 
-- **Encourage Experimentation**
-    This one needs little explanation. Whether you're helping create cslib, just using it your games, all that matters is that you're enjoying the experience.
+## Use It!
 
-## Overview 
+### Installation
+- Copy the file to your game folder together with your scene files.
```

Comment:
file**S** and I'd also suggest specifying only the files preceded with "cslib_".
E.g. "Copy all the 'cslib_' scene files to your game folder"